mlir/test/Dialect/SCF/parallel-loop-tiling-check-tile-size.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: mlir-opt %s -pass-pipeline='builtin.module(func.func(scf-parallel-loop-tiling{parallel-loop-tile-sizes=0,0}))' -split-input-file -verify-diagnostics
+
+// XFAIL: *
```

Comment:
You can check the expected error message with `// expected-error @below{{error message}}`

Reply:
It looks like it isn't possible to use `expected-error` to check for an error at an `UnknownLoc`. If this is true, am I good to go forward with using `XFAIL`?

Reply:
I didn't know that. Sounds good!